airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -786,7 +787,7 @@ private ConfiguredAirbyteCatalog getConfiguredCatalogWithOneStream(final String
     return catalog;
   }
 
-  private AirbyteCatalog getCatalog(final String defaultNamespace) {
+  protected AirbyteCatalog getCatalog(final String defaultNamespace) {
```

Comment:
why are these changed to protected? we should keep private unless required to change

Reply:
We need to override it in CockroachDbJdbcSourceAcceptanceTest. Basic Catalog contains a case with the table without PK meanwhile the Cockroach DB is not supposed to have a tables without PK. If you do not set it explicitly - it will anyway automatically add additionally.